scripts/tasks/sandbox-parts.ts

```diff
@@ -100,7 +92,7 @@ export const install: Task['run'] = async ({ sandboxDir }, { link, dryRun, debug
 
     await exec(
       'yarn install',
-      { cwd },
+      { cwd, stdout: 'inherit' },
       {
         dryRun,
```

Comment:
you should pass `debug` flag here, instead of `stdout` on line 95. Then if the command is run with `--debug`, it will print the full output like how the others behave